extensions/tiled-assets/examples/example-dimension.json

```
@@ -0,0 +1,239 @@
+{
+    "stac_version": "0.8.0",
```

```suggestion
    "stac_version": "0.9.0",
```
I guess this should be updated to be "0.9.0".We are still in 0.9.0-rc.1, but I guess it won't get into STAC before 0.9.0.

---

👍